parser2/error.go

```diff
@@ -21,15 +21,17 @@ func NewUnexpectedTokenError(tok Token, expected ...TokenType) error {
 
 func (u UnexpectedTokenError) Error() string {
```

Comment:
Lass uns mal auf ein Member/Interface-Getter-Format einigen, damit wir leichter "rusty"-Fehlermeldungen rendern können. Vgl. error.go und pos.go in https://github.com/golangee/architecture/tree/master/arc/token (kannst du gern rüber copy-pasten).

Reply:
Ich habe jetzt den [PosError aus token/error.go genommen](https://github.com/golangee/tadl/blob/4a545f058572ebf2cfa3dd159ecdaf353d9b051a/token/error.go#L42), den hast du im Lexer auch an einigen Stellen schon verwendet. Jetzt wird er überall benutzt.